packages/discord.js/src/managers/GuildBanManager.js

```diff
@@ -150,7 +153,10 @@ class GuildBanManager extends CachedManager {
     const id = this.client.users.resolveId(user);
     if (!id) throw new Error(ErrorCodes.BanResolveId, true);
     await this.client.rest.put(Routes.guildBan(this.guild.id, id), {
-      body: { delete_message_days: options.deleteMessageDays },
+      body: {
+        delete_message_days: options.deleteMessageDays,
+        delete_message_seconds: options.deleteMessageSeconds,
+      },
```

Comment:
I would rather just transform `delete_message_days` to `delete_message_seconds`. Something like this:

```JavaScript
  const deleteMessageSeconds = typeof options.deleteMessageDays !== 'undefined' ? options.deleteMessageDays * 172800 : options.deleteMessageSeconds;

  await this.client.rest.put(Routes.guildBan(this.guild.id, id), {
    body: { delete_message_seconds: deleteMessageSeconds },
    reason: options.reason,
  });
```

Reply:
I think you should do 24 * 60 * 60 instead of that magic number

Reply:
Edited that in. I think that'd look clearer.